email.js

```diff
@@ -19,8 +23,8 @@ var transporter = nodemailer.createTransport (
         }
     }
 );
-// email trigger
-cron.schedule('01 00 09 * * Sunday', () => {
+// email trigger 
+cron.schedule('01 35 15 * * Sunday', () => {
```

Comment:
Is there a way to manually trigger an email during our demo?  If this functionality is working we should show it off to the TAs.